docs/api/expressions/timestamps.md

```diff
@@ -6,15 +6,164 @@ All temporal operations are valid for both scalars and columns.
 ::: ibis.expr.types.temporal.TemporalValue
     options:
       show_source: true
+
+
 ::: ibis.expr.types.temporal.TimestampValue
     options:
       show_source: true
+      members: false
+<!-- prettier-ignore-end -->
+
+### Functions
+
+<!-- prettier-ignore-start -->
+::: ibis.expr.types.temporal.TimestampValue.add
```

Comment:
I wonder if it's actually less work to duplicate the method definitions in the class. I think they're all one-liners and there's only two of them.

Reply:
it ends up looking pretty funky, because `mkdocstrings` will pick up those two methods, place them  under a header called `Functions`, then the manually inserted ones will appear below, so you end up with

```
*** Functions
**** add
**** substract
*** Functions
**** between
...
...
```

I don't think we want to duplicate all of the mixin methods in each of these (cue conversation on whether code structure should be influenced by ability of documentation tool)

Reply:
Ugh, yeah okay, let's leave it as you have it!